backend/events.go

```diff
@@ -49,6 +58,21 @@ const (
 	Unknown                           //< Returned when the QueryContextCallback does not know how to deal with the given context.
 )
 
+// Add the InitCallback to the InitEvent to be called during initialization.
+// This should be called in a module's init() function.
+func (ie *InitEvent) Add(i InitCallback) {
+	*ie = append(*ie, i)
+}
+
+// Execute the InitEvent.
```

Comment:
Well, being an [un-exported method](http://golang.org/ref/spec#Exported_identifiers) it'll not end up in godoc anyway, but I think you are right that that seems to be the convention used. I hadn't paid attention to that before, but will try to keep in mind in the future.